.github/workflows/clean_caches.yml

```diff
@@ -58,7 +56,8 @@ jobs:
     if: github.event_name == 'pull_request'
     permissions: write-all
     runs-on: ubuntu-latest
-    steps:        
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
I don't know why this should be needed, but the workflow was running into permission issues when it ran the first time and the official docs seem to check it out.